wapitiCore/definitions/csp.py

```diff
@@ -24,6 +24,8 @@
 NAME = _("Content Security Policy Configuration")
 SHORT_NAME = _("CSP Configuration")
 
+WSTG_CODE = ["WSTG-CONF-12"]
+
```

Comment:
Please update OWASP reference to [https://owasp.org/www-project-web-security-testing-guide/latest/4-Web_Application_Security_Testing/02-Configuration_and_Deployment_Management_Testing/12-Test_for_Content_Security_Policy](https://owasp.org/www-project-web-security-testing-guide/latest/4-Web_Application_Security_Testing/02-Configuration_and_Deployment_Management_Testing/12-Test_for_Content_Security_Policy)